gui/src/pages/Offline.svelte

```diff
@@ -13,16 +13,18 @@
   });
 </script>
 
-<main>
-  <div>
-    <h3>
-      <code>exo</code> server is offline
-    </h3>
-    <p class="hint">
-      Run <Code>exo daemon</Code> in your terminal to start.
-    </p>
-  </div>
-</main>
+<Layout>
```

Comment:
I don't think that this should use the `Layout` because there are no actions possible while the server is offline, so navigation, etc. will be broken.

Reply:
`Layout` probably needs a more specific name. `PrimaryLayout` or `StandardLayout` or something like that?